includes/sdk-ts-browser/init.md

```diff
@@ -1,15 +1,29 @@
 You must initialize the SDK before you can instrument any events. Your Amplitude project's API key is required. You can pass an optional user ID and config object in this call. You can use the SDK anywhere after it's initialized anywhere in an application.
 
-```ts
-import { init }  from '@amplitude/analytics-browser';
+=== "Classic script"
 
-// Option 1, initialize with API_KEY only
-init(API_KEY);
+    ```ts
+    // Option 1, initialize with API_KEY only
+    amplitude.init(API_KEY);
 
-// Option 2, initialize including user ID if it's already known
-init(API_KEY, 'user@amplitude.com');
+    // Option 2, initialize including user ID if it's already known
+    amplitude.init(API_KEY, 'user@amplitude.com');
 
-// Option 3, initialize including configuration
-init(API_KEY, 'user@amplitude.com', configurationObj);
+    // Option 3, initialize including configuration
+    amplitude.init(API_KEY, 'user@amplitude.com', configurationObj);
+    ```
 
-```
+=== "Module script"
+
+    ```ts
+    import * as amplitude from '@amplitude/analytics-browser';
```

Comment:
Why classic vs module? Could we clarify installation methods up top (script tag vs npm) and then only show the "Classic script" in examples?

Reply:
Yeah, the classic and module might bring more confusion.

Reply:
You mean i'll also show how to import in installation config? That works too. I thought of being explicit just cause there was a brief time where we removed the import and we shortly got a question on how to import https://github.com/amplitude/Amplitude-TypeScript/issues/264

Reply:
I added the import code sample next to installation methods, but that's in quick start page, and not in the browser SDK page itself